### ISSUE TITLE
[Bug]: Garbage Nextcloud cannot handle thumbnails/previews in 2023 when using encryption. Awful.

### ISSUE THREAD

**Issue by Majestic7979** [nextcloud/desktop#6319]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

It's 2023. We've had many examples of ransomware out there. We have LetsEncrypt now providing anyone who wants to use TLS with a free and valid certificate to protect their websites. But Nextcloud treats file encryption as an afterthought, and if you enable it, kiss goodbye to file previews/thumbnails. It's like going back to the 90s and using Win95 "Windows Explorer" with just the icons. With the amount of multimedia files and PDFs being stored nowadays, this application is basically entirely useless. I could just set up a Samba share in my Linux server and spend some time setting up LUKS encryption and be done with Nextcloud forever. But I like having the ability to do AI search, that's why I use it.

It's about time the development team working on this application started taking usability more seriously instead of pointless UI changes. I don't give a rat's patootie if Nextcloud looks pretty. I just want it to be fast and to present my files in a usable way so I can find them quickly.

Also, the way the user has to go through file system to edit a config file to add preview file extensions to get Nextcloud to generate those previews is simply dumb. At least provide a way to add new extensions via the UI. Even when videos are added to the extensions the previews aren't generated.

<img width="1895" alt="image" src="https://github.com/nextcloud/desktop/assets/128995501/22a496ed-2dc3-4997-920b-76a1e1435d14">

### Steps to reproduce

Enable Nextcloud file encryption. Watch thumbnails not get generated for any of your files again.

### Expected behavior

As with any file manager or cloud storage sync client in 2023 and beyond I expect not to have a 1990s user experience, I expect to see file thumbnails with encryption enabled.

### Which files are affected by this bug

All files

### Operating system

Windows

### Which version of the operating system you are running.

Win 11

### Package

Other

### Nextcloud Server version

28.0.1

### Nextcloud Desktop Client version

3.11.0

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Enabled

### Are you using an external user-backend?

- [ ] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by Majestic7979** [comment]
Upon reading more about Nextcloud's useless encryption I decided to turn it off with occ command and then I also had to do a docker exec nextcloud apt update && docker exec nextcloud apt install ffmpeg which allowed thumbnail generation to run. I previously had ffmpeg installed but it seems like upon doing upgrades with watchtower the container will no longer have internal ffmpeg. This should be included IMO, as it's part of making essential functionality (previous/thumbnails) fully function.
